docs/auth/Single Sign-On via Device Browser.md

```diff
@@ -0,0 +1,27 @@
+## Single Sign-On (SSO) via the Device Browser
+The Auth SDK uses OpenID [AppAuth library](https://github.com/openid/AppAuth-Android) and implements OpenID's [auhtorization code flow](http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth) to achieve SSO via the device browser.
+
+### Prerequisite's
+1. Ensure your app is using the Android SDK - [Getting Started with AeroGear Services Android SDK](https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/getting-started.adoc) and add in the Auth SDK dependency - [AeroGear Services Auth SDD](https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/auth/README.adoc)
+2. Ensure that keycloak is preconfigured to manage authentication and access control for your service - [Keycloak Client Registration](http://www.keycloak.org/docs/3.2/securing_apps/topics/client-registration.html)
+
+### Steps to achieve SSO via device browser
+Let's assume that you have two apps, **Email App** and **Messaging App**.  Both apps have a service, Service A, that can only be accessed by authenticated users.  The authentication and access control to Service A should already be preconfigured in Keycloak (see prerequisite's).
```

Comment:
I think "Service" is a bit misleading. Realm might be a better choice of word.

Reply:
+1 to using Realm.  The apps can talk to as many services as they want as long as all of ther services are authorized using the same KC Realm.